2025-05-29 17:16:43,626 ERROR: Error fetching ink-business-preferred: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,627 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,629 INFO: Response status for citi-custom-cash: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,629 ERROR: Error fetching citi-custom-cash: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,629 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,630 INFO: Response status for capital-one-spark-cash-plus: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,630 ERROR: Error fetching capital-one-spark-cash-plus: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,630 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,631 INFO: Response status for bofa-premium-rewards: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,631 ERROR: Error fetching bofa-premium-rewards: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,631 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,632 INFO: Response status for amex-platinum: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,632 ERROR: Error fetching amex-platinum: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,632 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,635 INFO: Response status for capital-one-savor: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,635 ERROR: Error fetching capital-one-savor: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,635 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,636 INFO: Response status for amex-delta-reserve: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,636 ERROR: Error fetching amex-delta-reserve: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,636 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,637 INFO: Response status for capital-one-venture-rewards: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,637 ERROR: Error fetching capital-one-venture-rewards: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,638 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,638 INFO: Response status for amex-hilton-aspire: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,638 ERROR: Error fetching amex-hilton-aspire: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,638 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,639 INFO: Response status for chase-sapphire-preferred: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,639 ERROR: Error fetching chase-sapphire-preferred: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,639 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:43,663 INFO: Response status for citi-double-cash: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:43,663 ERROR: Error fetching citi-double-cash: 429 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:584]
2025-05-29 17:16:43,663 ERROR: Response content: {"message":"You have exceeded the rate limit per second for your plan, BASIC, by the API provider"} [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:585]
2025-05-29 17:16:44,156 INFO: Response status for wells-fargo-active-cash: 200 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:44,157 INFO: Successfully fetched data for wells-fargo-active-cash [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:581]
2025-05-29 17:16:44,160 INFO: Response status for discover-it-cash-back: 200 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:44,160 INFO: Successfully fetched data for discover-it-cash-back [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:581]
2025-05-29 17:16:44,207 INFO: Response status for amex-gold: 200 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:578]
2025-05-29 17:16:44,207 INFO: Successfully fetched data for amex-gold [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:581]
2025-05-29 17:16:44,207 INFO: Successfully fetched 3 out of 20 cards [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:635]
2025-05-29 17:19:15,688 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:303]
2025-05-29 17:20:39,487 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:303]
2025-05-29 17:21:27,633 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:303]
2025-05-29 17:21:27,660 INFO: Starting Hypercorn server... [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:764]
2025-05-29 17:22:55,151 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:303]
2025-05-29 17:22:55,172 INFO: Starting Hypercorn server... [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:764]
2025-05-29 17:23:02,319 ERROR: Plaid API error: Status Code: 400
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Server': 'nginx', 'Date': 'Thu, 29 May 2025 23:23:02 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '386', 'Connection': 'keep-alive', 'plaid-version': '2020-09-14', 'x-envoy-upstream-service-time': '54', 'x-envoy-decorator-operation': 'default.svc-apiv2:8080/*'})
HTTP response body: {
  "display_message": null,
  "documentation_url": "https://plaid.com/docs/?ref=error#invalid-input-errors",
  "error_code": "INVALID_PUBLIC_TOKEN",
  "error_message": "provided public token is in an invalid format. expected format: public-\u003cenvironment\u003e-\u003cidentifier\u003e",
  "error_type": "INVALID_INPUT",
  "request_id": "5T41R5SkgB2Xj7z",
  "suggested_action": null
}
 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:111]
2025-05-29 17:23:02,320 ERROR: Error fetching transactions: Status Code: 400
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Server': 'nginx', 'Date': 'Thu, 29 May 2025 23:23:02 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '386', 'Connection': 'keep-alive', 'plaid-version': '2020-09-14', 'x-envoy-upstream-service-time': '54', 'x-envoy-decorator-operation': 'default.svc-apiv2:8080/*'})
HTTP response body: {
  "display_message": null,
  "documentation_url": "https://plaid.com/docs/?ref=error#invalid-input-errors",
  "error_code": "INVALID_PUBLIC_TOKEN",
  "error_message": "provided public token is in an invalid format. expected format: public-\u003cenvironment\u003e-\u003cidentifier\u003e",
  "error_type": "INVALID_INPUT",
  "request_id": "5T41R5SkgB2Xj7z",
  "suggested_action": null
}
 [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:606]
2025-05-29 17:25:58,350 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:303]
2025-05-29 17:25:58,373 INFO: Starting Hypercorn server... [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:772]
2025-05-29 17:27:03,793 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:303]
2025-05-29 17:27:03,815 INFO: Starting Hypercorn server... [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:773]
2025-05-29 17:27:10,493 INFO: No public token provided, using sample transactions [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:614]
2025-05-29 17:27:10,494 INFO: Starting to fetch rewards data for 20 cards [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:641]
2025-05-29 17:27:10,525 ERROR: Error in fetch_all_card_rewards: name 'REWARDS_CC_BASE_URL' is not defined [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:699]
Traceback (most recent call last):
  File "/Users/jonathanhyatt/dinero/dinero-backend/app.py", line 653, in fetch_all_card_rewards
    results = await asyncio.gather(*tasks)
  File "/Users/jonathanhyatt/dinero/dinero-backend/app.py", line 567, in fetch_card_details
    if not all([REWARDS_CC_BASE_URL, REWARDS_CC_API_KEY, REWARDS_CC_API_HOST]):
NameError: name 'REWARDS_CC_BASE_URL' is not defined
2025-05-29 17:29:32,318 INFO: Application startup [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:322]
2025-05-29 17:29:32,341 INFO: Starting Hypercorn server... [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:792]
2025-05-29 17:29:40,697 INFO: No public token provided, using sample transactions [in /Users/jonathanhyatt/dinero/dinero-backend/app.py:633]
